src/renderer/api/api-manager.ts

```diff
@@ -46,12 +46,12 @@ export class ApiManager {
   @action
   registerStore(store: KubeObjectStore, apis: KubeApi[] = [store.api]) {
     apis.forEach(api => {
-      this.stores.set(api, store);
+      this.stores.set(api.kind, store);
```

Comment:
I think we cannot use just `kind` here because multiple apis might have the same kind.

Reply:
Switched to `apiBase` field. We consider it unique since storing `@observable apis` keys with same field
![registered apis](https://user-images.githubusercontent.com/9607060/107357409-0f323080-6ae3-11eb-9d9d-a87e33b1d878.png)